obs-to-maven

```diff
@@ -230,15 +283,19 @@ class Configuration:
             f = open(config_path, 'r')
             data = yaml.safe_load(f)
             f.close()
-        self.api = data.get('api', osc.conf.DEFAULTS['apiurl'])
+        self.api = data.get('api', "https://download.opensuse.org")
```

Comment:
Given that we don't use the OBS API anymore, `self.api` and the configuration entry `api` don't really fit anymore. As far as I understand this change, everyone needs to edit their configuration anyway¹, so this might be a good time to also change that. 

1: If "api" is set in the config

Reply:
right, `url` would be a better fit